File: pset1/mario.c
#include<cs50.h>
#include<stdio.h>

int main(void)
{
    int n;
    do
    {
        n = get_int("Enter the height: ");
    }
    while(n<1 || n>8);

    // Pattern Starting
    for(int i=1; i<=n; i++)
    {
        // Spaces
        for(int j=0; j<n-i; j++)
        {
            printf(" ");
        }

        // Left hashes
        for(int j=0; j<i; j++)
        {
            printf("#");
        }

        // Gap
        printf(" ");

        // Right hashes
        for(int j=0; j<i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: pset6/mario.py
from cs50 import get_int

while True:
    height = get_int("Enter the height: ")
    if int(height)>=1 and int(height)<=8:
        break

# Pattern
for i in range(1, height + 1):
    num_hashes = i
    num_spaces = height - num_hashes

    print(" " * num_spaces, end="")
    print("#" * num_hashes, end="")
    print("  ", end="")
    print("#" * num_hashes)